src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/EnumTypeModelBinderProvider.cs

```diff
@@ -10,11 +10,11 @@ namespace Microsoft.AspNetCore.Mvc.ModelBinding.Binders
     /// </summary>
     public class EnumTypeModelBinderProvider : IModelBinderProvider
     {
-        private readonly bool _allowBindingUndefinedValueToEnumType;
+        private readonly MvcOptions _options;
 
-        public EnumTypeModelBinderProvider(bool allowBindingUndefinedValueToEnumType)
+        public EnumTypeModelBinderProvider(MvcOptions options)
```

Comment:
Add a new ctor which accepts `MvcOptions` and keep the old ctor. You could mark it as obsolete.

Reply:
This type hasn't been released yet and has been part of 2.1.0-preview1, so didn't add another constructor.